enterprise/frontend/src/embedding-sdk/hooks/private/use-init-data.ts

```diff
@@ -25,6 +25,12 @@ interface InitDataLoaderParameters {
 export const useInitData = ({ config }: InitDataLoaderParameters) => {
   const { allowConsoleLog = true } = config;
 
+  // This is outside of a useEffect otherwise calls done on the first render could use the wrong value
+  // This is the case for example for the locale json files
+  if (api.basename !== config.metabaseInstanceUrl) {
```

Comment:
Without this, I noticed the app trying to load the font from the the fe url first, and then from mb